proto/public/api.proto

```diff
@@ -124,8 +124,68 @@ message ApiError {
   string error = 1;
 }
 
+message ResourceSpec {
+  string namespace = 1;
+  string resource = 2;
+}
+
+message ResourceSelection {
+  ResourceSpec spec = 1;
+  string label_selector = 2;
+}
+
+message ResourceError {
+  ResourceSpec spec = 1;
+  string error = 2;
+}
+
+message StatSummaryRequest {
+  ResourceSelection resource = 1;
+  TimeWindow time_window = 2;
+
+  oneof outbound {
+    Empty none = 3;
+    ResourceSpec out_to_resource   = 4;
+    ResourceSpec out_from_resource = 5;
+  }
+}
+
+message StatSummaryResponse {
+  oneof response {
+    Ok ok = 1;
+    ResourceError error = 2;
+  }
+
+  message Ok {
+    repeated StatTable stat_tables = 1;
+  }
+}
+
+message StatTable {
+  oneof table {
+    PodGroup pod_group = 1;
+  }
+
+  message PodGroup {
+    repeated Row rows = 1;
+
+    message Row {
```

Comment:
In general, this looks fine to me.

I think eventually we should try to make this even more generic so that we could add columns without having to change the code in the CLI at all, and so that we could even add new kinds of tables without changing the code in the CLI.

(I understand that right now we're optimizing for getting some specific things working, so this suggestion shouldn't be considered blocking any merge. We should consider whether we want to have a more generic data model for before releasing a stable 1.0 of Conduit.)